pkg/resolvers/endpoints.go

```diff
@@ -327,10 +327,14 @@ func (r *defaultEndpointsResolver) getMatchingPodAddresses(ctx context.Context,
 			r.logger.Info("pod IP not assigned yet", "pod", k8s.NamespacedName(&pod))
 			continue
 		}
-		portList := r.getPortList(pod, ports)
-		if len(ports) != len(portList) && len(portList) == 0 {
-			r.logger.Info("Couldn't get matched port list from the pod", "pod", k8s.NamespacedName(&pod), "expectedPorts", ports)
-			continue
+		var portList []policyinfo.Port
+		// no need to populate src pods' ports for ingress rules
```

Comment:
nit: Why do we even need this comment here? We're not looking in to src pods anyways

Reply:
The comment was removed in latest update.